ui/imports/shared/popups/ImportCommunityPopup.qml

```diff
@@ -1,82 +1,139 @@
 import QtQuick 2.12
 import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.14
 import QtGraphicalEffects 1.13
 import QtQuick.Dialogs 1.3
+import QtQml.Models 2.14
 
 import utils 1.0
 import shared.controls 1.0
 
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
-import StatusQ.Popups 0.1
-import StatusQ.Controls 0.1 as StatusQControls
+import StatusQ.Popups.Dialog 0.1
+import StatusQ.Controls 0.1
 
-StatusModal {
+StatusDialog {
     id: root
-    width: 640
-    height: 400
-
     property var store
+    property bool isPrivateKey: false
+    property bool isPublicKey: false
+    property string errorMessage: ""
```

Comment:
Is this an API of the dialog? Or is it private and should be hidden under `QtObject { id: d }`?

Reply:
This should probably live in QtObject, now that you mention it!